modules/images/regenerate-existing-images/background-process/class-perflab-background-process.php

```diff
@@ -0,0 +1,236 @@
+<?php
+/**
+ * Background process runner class.
+ *
+ * @since n.e.x.t
+ * @package performance-lab
+ */
+
+/**
+ * Class Perflab_Background_Process.
+ *
+ * Runs the heavy lifting tasks in background in separate process.
+ */
+class Perflab_Background_Process {
+	/**
+	 * Name of the action which will trigger background
+	 * process to run the job.
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @var string Background process action name.
+	 */
+	const BG_PROCESS_ACTION = 'background_process_handle_request';
```

Comment:
@ankitrox we should prefix this action name with `perflab_`
```suggestion
	const BG_PROCESS_ACTION = 'perflab_background_process_handle_request';
```